src/styles/frederative.js

```diff
@@ -169,7 +171,12 @@ export default class frederativeStyle extends Style {
     this._p5.pop();
   }
 
-  afterDraw() { this.dither(null); }
+  afterDraw() { 
+    this.dither(null); 
+
+    // reset user's density after my dithering is done
+    this._p5._pixelDensity = this.currDensity;
```

Comment:
This is just the internal p5 state, p5.resizeCanvas() still needs to be called to update the actual drawing context. 
Calling this._p5.pixelDensity(this.currDensity) does this for us but I see that it also clears your drawing :(
I think the easiest additional change is to also update index.js' draw method to call p5.resizeCanvas(s, s, true) at the beginning where it resets colors and other things. 
p5 code for reference: https://github.com/processing/p5.js/blob/v1.4.1/src/core/environment.js#L598

Reply:
i added this fix and it *seems* to work -- i just made a new commit and will PR shortly.

Reply:
Verified that it's fixed after your update, https://fxhash-bridge-mmixbxwe4-camilleroux.vercel.app/.  Thank you!